siriuspy/tests/test_util.py

```diff
@@ -4,12 +4,13 @@
 
 from io import StringIO
 import logging
-import numpy as np
 from unittest import mock, TestCase
+import numpy as np
+
 import siriuspy.util as util
 
 
-public_interface = (
+PUB_INTERFACE = (
```

Comment:
Should I convert `PUBLIC_INTERFACE` in all modules to simpler `PUB_INTERFACE`? Use of editors find-and-replace is much simpler if we keep standard as much as possible.

Reply:
I think it's a good idea. but let me do it, I want to use `sed`! :joy:

Reply:
done!

Reply:
nice. _grep_, _sed_, _code_... you choose your arm!